glib/src/variant.rs

```diff
@@ -1218,7 +1218,11 @@ impl ToVariant for std::ffi::OsString {
 #[cfg(unix)]
```

Comment:
The docs above here and also below need to be updated

Reply:
But these docs only want about portability. That comment is there even for `any(unix,windows)`, so I guess these now should be just that.

Reply:
They're not really platform-specific anymore but according to the GLib rules. Before there was the `u8` / `u16` difference, now they're just byte arrays in either case and work the same like everything else in GLib on all platforms.

So I think either these portability notes should be removed, or maybe there should be a note of the above (GLib rules, local encoding on UNIX, UTF-8 on Windows).